docs/source/dataset_summary.rst

```diff
@@ -17,25 +17,25 @@ Motor Imagery
 ======================
 
 .. csv-table::
-   :header: Dataset, #Subj, #Chan, #Classes, #Trials, len, Sampling rate, #Sessions, #Trials*#Sessions
+   :header: Dataset, #Subj, #Chan, #Classes, #Trials, Trial length, Freq, #Session, #Runs, Total_trials
    :class: sortable
 
-   AlexMI,8,16,3,20,3s,512Hz,1,20
-   BNCI2014001,10,22,4,144,4s,250Hz,2,288
-   BNCI2014002,15,15,2,80,5s,512Hz,1,80
-   BNCI2014004,10,3,2,360,4.5s,250Hz,5,1800
-   BNCI2015001,13,13,2,200,5s,512Hz,2,400
-   BNCI2015004,10,30,5,80,7s,256Hz,2,160
-   Cho2017,53,64,2,100,3s,512Hz,1,100
-   Lee2019_MI,55,62,2,100,4s,1000Hz,2,200
-   MunichMI,10,128,2,150,7s,500Hz,1,150
-   Schirrmeister2017,14,128,4,120,4s,500Hz,1,120
-   Ofner2017,15,61,7,60,3s,512Hz,1,60
-   PhysionetMI,109,64,4,23,3s,160Hz,1,23
-   Shin2017A,29,30,2,30,10s,200Hz,3,90
-   Shin2017B,29,30,2,30,10s,200Hz,3,90
-   Weibo2014,10,60,7,80,4s,200Hz,1,80
-   Zhou2016,4,14,3,160,5s,250Hz,3,480
+        AlexMI,8,16,3,20,3s,512Hz,1,1,480
+        BNCI2014001,9,22,4,144,4s,250Hz,2,6,62208
+        BNCI2014002,14,15,2,80,5s,512Hz,1,8,17920
+        BNCI2014004,9,3,2,360,4.5s,250Hz,5,1,32400
+        BNCI2015001,12,13,2,200,5s,512Hz,3,1,14400
+        BNCI2015004,9,30,5,80,7s,256Hz,2,1,7200
+        Cho2017,49,64,2,100,3s,512Hz,1,1,9800
```

Comment:
Cho2017 must have 52 since we have put the modification in https://github.com/NeuroTechX/moabb/pull/392